src/api/users/src/services/firestore.js

```diff
@@ -1,23 +1,41 @@
 const admin = require('firebase-admin');
 const { logger } = require('@senecacdot/satellite');
+const fs = require('fs');
+const path = require('path');
 
-const {
-  FIREBASE_TYPE,
-  FIREBASE_PROJECT_ID,
-  FIREBASE_PRIVATE_KEY_ID,
-  FIREBASE_PRIVATE_KEY,
-  FIREBASE_CLIENT_EMAIL,
-  FIREBASE_CLIENT_ID,
-  FIREBASE_AUTH_URI,
-  FIREBASE_TOKEN_URI,
-  FIREBASE_AUTH_PROVIDER_X509_CERT_URL,
-  FIREBASE_CLIENT_X509_CERT_URL,
-} = process.env;
+let FIREBASE_TYPE;
```

Comment:
I don't think you need all these.  We should be `require()`ing the firebase JSON file, which will define all these values, and we just pass that object through.